src/app/(main)/profile/reservations/@modal/(.)confirm/[reservationId]/page.tsx

```
@@ -0,0 +1,21 @@
+'use client';
+import ConfirmModal from '@/components/common/ConfirmModal';
+import { cancelReservation } from '@/lib/api/profile/reservationList';
+import { useRouter, useParams } from 'next/navigation';
+
+const ConfirmModalPage = () => {
+  const router = useRouter();
+  const params = useParams();
+  const reservationId = params.reservationId;
+  const onDismiss = async () => {
+    const res = await cancelReservation(Number(reservationId));
```

ESLint

[error] 'res' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)